flask_ext_migrate/__init__.py

```diff
@@ -42,33 +42,36 @@ def fix_from_imports(red):
     for node in from_imports:
         modules = node.value
 
-        if (len(modules) < 2 or
-                modules[0].value != 'flask' or modules[1].value != 'ext'):
-                    continue
+        if modules[0].value != 'flask' and modules[1].value != 'ext':
```

Comment:
I think this may break simple imports such as `import requests`, could you add a test for that?